doc/01-getting-started.md

```diff
@@ -60,15 +60,15 @@ to the `stack.yaml` file:
 
 ```yaml
 extra-deps:
-- aws-lambda-haskell-runtime-2.0.1
+- aws-lambda-haskell-runtime-3.0.0
```

Comment:
This assumes that `aws-lambda-haskell-runtime-3.0.0` is up on Hackage.

To test it out currently, use the following

```yaml
extra-deps:
- git: https://github.com/dnikolovv/aws-lambda-haskell-runtime.git
  commit: 1cc5a39dad2b111d05e7cb6496571ad97a8cca59
```